### PR TITLE
Move to React

### PR DESCRIPTION
**Pull request by bpierre**
Note: this PR also includes the color system.

- [x] Port the structure, webpack config etc. to React
- [x] Color palettes
- [x] Gallery
- [x] Button
- [x] Colors
- [x] Theme
- [x] Text
- [x] Header
- [x] Footer
- [x] Section
- [x] PreFooter
- [ ] <s>Loader</s>

### CODE REVIEW

**Comment by onbjerg** [comment]
That's a big PR 😵

**Comment by bpierre** [comment]
Yeah, better not looking at the diff, most of the things have changed ha ha

**Comment by onbjerg** [comment]
Btw, is there a specific reason we're building our own gallery instead of using [storybook](https://github.com/storybooks/storybook)?

**Comment by bpierre** [comment]
Yes! Sorry for not having explained this before.

The objective I have in mind is to release a documentation website (looking like an Aragon website), also providing the design specifications.

Eventually, I’d like to provide to Aragon developers a unified website, containing everything needed to build on the platform: node-aragon API, Aragon UI components API, but also guides and guidelines.

Storybook provides a super nice environment to develop components (it’s my go-to solution :heart:), but I don’t think it would be flexible enough for our case (or it would mean disabling its main features like the bottom panel).

To summarize: more control, trivial to implement (menu + routing + stories), dogfooding 🐶, and eventually a better experience.

**Comment by bpierre** [comment]
OK, this is ready to be merged now.

`Loader` is the only component missing, I think it’s too specific for now. I’d like to see how we manage loading states in the Aragon client before adding a generic `Loader` component.
